### PR TITLE
Creating a data model milestone

### PR DESCRIPTION
**Pull request by ghost**
For this milestone, the requirements were:

Build your project schema.

    Create and run the necessary migration files.
    Table and column names should match the ERD diagram.
        note: photo for users table should be a link to an image
    Foreign keys should be included.
    All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

File: db/migrate/20220603195131_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.text :photo
```
Comment (AshakaE) [inline db/migrate/20220603195131_create_users.rb]: - [x] kindly change this datatype to string because it's more likely to hold a url to an image

**Review by AshakaE — CHANGES_REQUESTED** [review]
Hi @iKuartz , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20220603195131_create_users.rb (hunk 1 above)
Comment (ghost) [reply db/migrate/20220603195131_create_users.rb]: Ok, thank you.

**Review by AshakaE — APPROVED** [review]
Hi @iKuartz , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**